models/datatypes/NcMethodResultLength.json

```
@@ -4,10 +4,10 @@
   "type": 2,
   "fields": [
     {
-      "description": "Length result value",
+      "description": "Length result value. Can be null if the underlying sequence is null",
```

As above for consistency(?):
```suggestion
      "description": "Sequence length result value. MUST be null if the sequence is null",
```